app/models/concerns/twitter_user/api.rb

```diff
@@ -0,0 +1,262 @@
+require 'active_support/concern'
+
+module Concerns::TwitterUser::Api
+  extend ActiveSupport::Concern
+
+  included do
+    def one_sided_following
+      ExTwitter::Client.new.one_sided_following(self)
+    end
+
+    def one_sided_followers
+      ExTwitter::Client.new.one_sided_followers(self)
+    end
+
+    def mutual_friends
+      ExTwitter::Client.new.mutual_friends(self)
+    end
+
+    def common_friends(other)
+      return [] if other.blank?
+      ExTwitter::Client.new.common_friends(self, other)
+    end
+
+    def common_followers(other)
+      return [] if other.blank?
+      ExTwitter::Client.new.common_followers(self, other)
+    end
+
+    def removing
+      return [] if TwitterUser.where(uid: uid, user_id: user_id).limit(2).pluck(:id).size < 2
+      TwitterUser.where(uid: uid, user_id: user_id).order(created_at: :asc).each_cons(2).map do |old_one, new_one|
+        ExTwitter::Client.new.removing(old_one, new_one)
+      end.flatten
+    end
+
+    def removed
```

Comment:
__[rubocop]__
* Assignment Branch Condition size for removed is too high. [16.03/15] [:ref](http://c2.com/cgi/wiki?AbcMetric)